bindings/tpython/src/TPython.cxx

```diff
@@ -167,7 +162,7 @@ Bool_t TPython::Import(const char *mod_name)
 
    // allow finding to prevent creation of a python proxy for the C++ proxy
    Py_INCREF(mod);
-   PyModule_AddObject(PyROOT::gRootModule, mod_name, mod);
+   PyModule_AddObject(CPyCppyy::PyStrings::gThisModule, mod_name, mod);
```

Comment:
`PyROOT::gRootModule` is a Python module, but `CPyCppyy::PyStrings::gThisModule` is a Python string. This replacement is therefore wrong, we should put `CPyCppyy::gThisModule`.